app/state.go

```diff
@@ -645,6 +645,10 @@ func (u *updateCheckState) Handle(ctx *StateContext, c Controller) (State, bool)
 
 	update, err := c.CheckUpdate()
 
+	// Insert the UpdateControlMap into the pool if present
```

Comment:
The other alternative is doing it here. I vied for `mender.go`, but all ears here :ear: